morebits.js

```diff
@@ -2676,29 +2678,37 @@ Morebits.wiki.page = function(pageName, currentAction) {
 	 *
 	 * Referred to as "review" on-wiki
 	 *
-	 * Will, by it's nature, mark as patrolled as well.
+	 * Will, by it's nature, mark as patrolled as well. Falls back to
+	 * patrolling if not in an appropriate namespace.
 	 *
 	 * Doesn't inherently rely on loading the page in question; simply
 	 * passing a pageid to the API is sufficient, so in those cases just
 	 * using Morebits.wiki.api is probably preferable.
 	 *
+	 * Will first check if the page is queued via fnProcessTriageList
+	 *
 	 * No error handling since we don't actually care about the errors
 	 */
 	this.triage = function() {
-		if (!Morebits.userIsSysop && !Morebits.userIsInGroup('patroller')) {
-			return;
-		}
-
-		// If on the page in question, don't need to query for page ID
-		if (new mw.Title(Morebits.pageNameNorm).getPrefixedText() === new mw.Title(ctx.pageName).getPrefixedText()) {
-			ctx.pageID = mw.config.get('wgArticleId');
-			fnProcessTriage(this, this);
+		// Fall back to patrol if not a valid triage namespace
+		if ([0, 2, mw.config.get('wgPageTriageDraftNamespaceId')].indexOf(mw.config.get('wgNamespaceNumber')) === -1) {
```

Comment:
use `mw.config.get('pageTriageNamespaces')` instead.

Reply:
Hah yes!  I thought I was crazy not seeing it, serves me right for checking `wg`-prefixed items...